qtpy/QtWidgets.py

```diff
@@ -14,12 +14,24 @@
 
 _missing_optional_names = {}
 
+
 def __getattr__(name):
     """Custom getattr to chain and wrap errors due to missing optional deps."""
     raise getattr_missing_optional_dep(
         name, module_name=__name__, optional_names=_missing_optional_names)
 
 
+def _possibly_static_exec(cls, *args, **kwargs):
+    """ Call `self.exec` when `self` is given or a static method otherwise. """
```

Comment:
```suggestion
    """Call `self.exec` when `self` is given or a static method otherwise."""
```

Nit: Strip excess white space per PEP 257.